server/src/authorizers/permissions.js

```diff
@@ -0,0 +1,20 @@
+const { shield } = require('graphql-shield');
+
+const { isChannelAllowed } = require('./channels');
+
+function getPermissions() {
+  return shield({
+    Query: {
+      measurement: isChannelAllowed,
+      measurements: isChannelAllowed,
+      measurable: isChannelAllowed,
+      measurables: isChannelAllowed,
+      series: isChannelAllowed,
+      seriesCollection: isChannelAllowed,
```

Comment:
This feels like the wrong way to do things. I'd prefer it if a user could query for all measurables/measurements, but only get back ones that belong to all the channels they have access to. They shouldn't need to declare one specific channel upfront.

Reply:
Fixed.